README.md

```diff
@@ -125,6 +126,12 @@ Go to <https://github.com/JuliaLang/VersionsJSONUtil.jl/actions/workflows/CI.yml
 click "Run workflow," ensure "Branch: main" is selected, then hit the button.
 This takes a bit under 2 hours to complete.
 
+### Building the PDF documentation
+
+Go to https://github.com/JuliaLang/docs.julialang.org/actions/workflows/PDFs.yml and
+click "Run workflow", ensure "Branch: master" is selected, and then hit the button.
+This takes at least 1 hour to complete.
```

Comment:
```suggestion
This takes at least 1 hour to complete.

In case the PDF fails to build on CI, but can still be compiled locally, the locally produced PDF can
manually be pushed to the [`assets` branch](https://github.com/JuliaLang/docs.julialang.org/tree/assets)
of the repository
```